src/content/docs/guides/policy-diff.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Reviewing Policy
+description: How to review LavaMoat Policy and its diffs
+---
+
+<!-- markdownlint-disable no-inline-html -->
+
+This guide will help you review your policy on a continuous basis.
+
+## Why review Policy?
+
+The Policy File generated by LavaMoat is based on a scan of your codebase, identifying all the powers it uses. The initial policy doesn't provide security on its own. Instead, it's your review of the initial policy and the subsequent updates (with new or updated dependencies) that makes your application secure.
+
+Reviewing diffs as you go lets you spot suspicious packages or limit the powers you wish newly added packages can use.
+
+The review of your initial policy may seem like a lot of effort, but it's an investment in a good starting point for your application.  
```

Comment:
```suggestion
The review of your initial policy may seem like a big investment, but it's  a good starting point for your application.  
```

Reply:
I intend to say "investment" as a good thing. If optimizing for brevity, this sentence is redundant. It's here to motivate. Yes, you need to put work in, but it pays off.